style/listview.less

```diff
@@ -77,5 +77,28 @@
 			font-size: 17px;
 			color: #323232;
 		}
+
+		.empty-search {
+			margin-top: 65px;
+			font-size: 17px;
+			font-stretch: normal;
+			font-style: normal;
+			line-height: normal;
+			letter-spacing: normal;
+			text-align: center;
+			color: #323232;
+		}
+
+		.empty-search-explainer {
+			font-size: 14px;
+			font-weight: normal;
+			font-stretch: normal;
+			font-style: normal;
+			line-height: normal;
+			letter-spacing: normal;
+			text-align: center;
+			color: #72777d;
+			padding: 7px 35px 0 35px;
+		}
```

Comment:
The styling breaks in landscape mode, and the case when there is no explainer [1]

[1]

<img src="https://user-images.githubusercontent.com/2560096/77394358-6d590680-6d9f-11ea-8a1a-0f65791e5fde.png" width="200" >

---

✨ The initial implementation of the `ListView` and `RadioListView` are designed for the general purpose used, therefore it accepts `{empty}` be text or any component for any customized empty content. That's fine for now as we can work for another refactor pr and also let Radio be within `ListView`

Reply:
> the case when there is no explainer [1]

Huei could you please list out steps to get to that state? I'm not able to replicate

Reply:
Currently there is only search page uses `ListView` with `empty` message key, so when I remove `emptySubText` prop in Search page, then I got the above screenshot.